pmm-tests/pmm-2-0-bats-tests/helm-test.bats

```diff
@@ -0,0 +1,111 @@
+## pmm k8s helm tests
+### needs: helm, kubectl, k8s cluster, default kubeconfig
+
+setup() {
+
+    PROJECT_ROOT=$(dirname "$BATS_TEST_FILENAME")
+
+    echo "file name $BATS_TEST_FILENAME"
+
+    source "$PROJECT_ROOT/k8s_helper.sh"
```

Comment:
I believe we assume user has already started the local cluster on their system

Reply:
yes, look at first comment :)